src/components/Filters.jsx

```diff
@@ -0,0 +1,143 @@
+                        <label htmlFor="genre-classics" className="text-sm genre-label flex-row flex-align-center">
+                            <input type="checkbox" className="input-checkbox" id="genre-classics" />
+                            <span className="genre-text">
+                                Classics
+                            </span>
+                        </label>
+                    </li>
+                    <li className="list-item mb-0-25  p-0-25">
+                        <label htmlFor="genre-mystery" className="text-sm genre-label flex-row flex-align-center">
+                            <input type="checkbox" className="input-checkbox" id="genre-mystery" />
+                            <span className="genre-text">
+                                Mystery
+                            </span>
+                        </label>
+                    </li>
+                    <li className="list-item mb-0-25 p-0-25">
+                        <label htmlFor="genre-fantasy" className="text-sm genre-label flex-row flex-align-center">
+                            <input type="checkbox" className="input-checkbox" id="genre-fantasy" />
+                            <span className="genre-text">
+                                Fantasy
+                            </span>
+                        </label>
+                    </li>    
+                </ul>        
+            </div>
+            <div className="filter-price mt-1 mb-2 filter">
+                <ul className="list list-stacked list-style-none mt-1 mb-1-5 filter-list">
+                    <label htmlFor="product-price-slider">          
+                        <h6 className="filter-head text-reg list-head px-0-5 py-0-25 my-0-5">
+                            Price
+                        </h6>
+                        <input type="range" className="input-range" id="product-price-slider" />
+                    </label>
+                    <div className="slider-price-range flex-row flex-justify-between flex-align-center">
+                        <span className="text-sm min-price">₹ 100</span>
+                        <span className="text-sm min-price">₹ 10,000</span>
+                    </div>
+                </ul>   
+            </div>
+            <div className="filter-ratings mt-1 mb-2">
+                <ul className="list-spaced list-style-none">
+                    <h6 className="filter-head text-reg list-head px-0-5 py-0-25 my-0-5 mb-0-5">Ratings</h6>
+                    <li className="list-item mt-0-75 mb-0-5">
+                        <label htmlFor="rating-4-above" className="text-sm rating-label flex-row flex-align-center">
+                            <input type="radio" className="input-radio"  id="rating-4-above" name="product-ratings"/>
+                            <span className="genre-text">
+                                4 stars &amp; above
+                            </span>
+                        </label>
```

Comment:
You can create an array for ratings and loop through it. The option for rating will not increase in future but just to reduce the number of lines of code. This is absolutely fine too.

Reply:
Yes, this makes sense. I shall make this change once I'm done with the basic P0 requirements!